CHANGELOG.asciidoc

```diff
@@ -29,6 +29,8 @@ https://github.com/elastic/beats/compare/v5.3.0...master[Check the HEAD diff]
 *Affecting all Beats*
 - Fix console output {pull}4045[4045]
 
```

Comment:
a space here, but you will have to cleanup the changelog anyways in the other PR.